apiv2/pkg/clients/robot/robot.go

```diff
@@ -87,14 +87,31 @@ func (in AccessAction) String() string {
 
 // ListRobotAccounts ListProjectRobots returns a list of all robot accounts.
 func (c *RESTClient) ListRobotAccounts(ctx context.Context) ([]*modelv2.Robot, error) {
-	resp, err := c.V2Client.Robot.ListRobot(&robot.ListRobotParams{
-		Context: ctx,
-	}, c.AuthInfo)
-	if err != nil {
-		return nil, handleSwaggerRobotErrors(err)
+	var robotAccounts []*modelv2.Robot
+	var page int64 = c.Options.Page
+	var pageSize int64 = c.Options.PageSize
+
+	for {
+		resp, err := c.V2Client.Robot.ListRobot(&robot.ListRobotParams{
+			Context:  ctx,
+			Page:     &page,
+			PageSize: &pageSize,
+		}, c.AuthInfo)
```

Comment:
```suggestion
	resp, err := c.V2Client.Robot.ListRobot(&robot.ListRobotParams{
		Page:     &c.Options.Page,
		PageSize: &c.Options.PageSize,
		Q:        &c.Options.Query,
		Sort:     &c.Options.Sort,
		Context:  ctx,
	}, c.AuthInfo)
```

The wrapping `for`-loop could then be removed.